clients/fossid-webapp/src/main/kotlin/model/result/Snippet.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2023 The ORT Project Authors (See <https://github.com/oss-review-toolkit/ort-server/blob/main/NOTICE>)
```

Comment:
"See" -> "see"

Reply:
And the URL needs to be https://github.com/oss-review-toolkit/ort/blob/main/NOTICE.